runtime/jcl/exports.cmake

```diff
@@ -596,6 +595,7 @@ endif()
 # java 15+
 if(NOT JAVA_SPEC_VERSION LESS 15)
```

Comment:
Not consistent with `JAVA_SPEC_VERSION >= 14`, which is used with `Class.getRecordComponentsImpl`'s implementation? May be, it is not critical since we do not support Java 12-16 anymore.

Reply:
The minimum version of cmake we accept doesn't have `LESS_EQUAL` so we can't write `JAVA_SPEC_VERSION LESS_EQUAL 14`: `NOT JAVA_SPEC_VERSION LESS 15` *is* equivalent to `JAVA_SPEC_VERSION >= 14`.